etlas/Distribution/Client/Install.hs

```diff
@@ -1369,15 +1369,17 @@ installLocalTarballPackage verbosity pkgid
       let relUnpackedPath = if isGit then ""
                             else display pkgid ++ (if isBinary then "-bin" else "")
           absUnpackedPath = tmpDirPath </> relUnpackedPath
-          descFilePath = absUnpackedPath
-                     </> display (packageName pkgid) <.> "cabal"
+          descFilePath ext = absUnpackedPath
+                         </> display (packageName pkgid) <.> ext
+          dhallFile = absUnpackedPath </> "etlas.dhall"
       info verbosity $ (if isGit then "Copying " else "Extracting ")
                     ++ tarballPath
                     ++ " to " ++ tmpDirPath ++ "..."
       patchedExtractTarGzFile verbosity False tmpDirPath relUnpackedPath tarballPath patchesDir isGit isBinary
-      exists <- doesFileExist descFilePath
-      when (not exists) $
-        die' verbosity $ "Package .cabal file not found: " ++ show descFilePath
+      existsAny <- forM [descFilePath "etlas", descFilePath "cabal", dhallFile]
+                   (doesFileExist . descFilePath)
```

Comment:
Why is `descFilePath` called twice - once inside the first argument of `forM` and again in the second argument of `forM`?

Reply:
Whoops, what a bug, good catch
I'll fix it asap